### DISCUSSION TITLE
Welcome to wdio-rerun-service Discussions!

### DISCUSSION THREAD

**Discussion by esaari** [webdriverio-community/wdio-rerun-service#62] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by asalmanov** [comment]
Hi, 
Is there a way to rerun only the failing scenario with wdio cucumber? If possible please give details step by step. Thanks in advance.

**Reply by mikesalvia** [reply]
That is exactly what this service allows.

After configuring and running there should be a 'rerun.sh' file in the root of your project.

Are you referring to just the failed and not broken or skipped? That feature has not yet been implemented.

**Reply by nimeshbhatt** [reply]
Hi @mikesalvia, 

I am able to generate the .json and .sh file successfully. 

I could see the details within .sh file which is,

`DISABLE_RERUN=true node_modules/.bin/wdio run   --spec=C:/Users/{{profile}}/Documents/Projects/Development/features/ login.feature:12`

But, I am not able to run this .sh file. I am on my windows machine and tried to execute .sh file via below command,
`bash rerun.sh`.  It is throwing an error `rerun.sh: line 1: ./node_modules/.bin/wdio: No such file or directory`

Can you please help me how can I execute my rerun.sh file

**Comment by willycedric** [comment]
Hello Everyone, 
Please could we have an example on how to write the rerun.sh script ? 
Thx
